packages/guesstimator/formatter/formatters/Function.js

```diff
@@ -1,8 +1,15 @@
 import _ from "lodash";
+import {shorthandIntoLognormalFormattingStep} from './lib'
+
 //FIXME
+
+// The function: shorthandIntoLognormalFormattingStep
+// transforms strings like "=mm(normal(10,5), 1 to 100))
+// into strings like "=mm(normal(10,5), lognormal(50.1, 1.4))
```

Comment:
This to me is a fairly useful comment, as it makes the interface straightforward. However, I think it should be in the code for shorthandIntoLognormalFormattingStep, not here, where it's used. It could be used in several places later.